docs/design/simple-rolling-update.md

```diff
@@ -0,0 +1,37 @@
+## Simple rolling update
+This is a lightweight design document for simple rolling update in ```kubectl```
+
+### Lightweight rollout
+Assume that we have a current replication controller named ```foo``` and it is running image ```image:v1```
```

Comment:
This assumes a single container, obviously.

As much as possible, flags should match kubectl run/run-container.

If we were to support multiple containers in run, we would support that here, also.